### PR TITLE
feat(edge): v1.3

### PR DESCRIPTION
**Pull request by DannyS03**
## Context

Release documentation for Edge v1.3.

- Introduces proxy-admin-address with the reintegration of proxy contracts for upgradability.
- Introduces baseFeeChangeDenom.
- Updates performance benchmarks.
- Updates flag tables.
- General presentation and formatting updates.

### CODE REVIEW

File: docs/edge/operate/deploy/access-control/allowlist-general.md [hunk 1]
```diff
@@ -48,6 +48,21 @@ To interact with the ACL precompiles using an external library or client, follow
...
+```shell
+function setAdmin(address)
+function setEnabled(address)
+function setNone(address)
+function readAddressList(address) returns (uint256)
+function setListEnabled(bool)
+function getListEnabled() returns (bool)
+```
+
+<!-- to document interaction -->
```
Comment (Stefan-Ethernal) [inline docs/edge/operate/deploy/access-control/allowlist-general.md:64]: Are you going to add the code snippets through this PR?

File: docs/edge/operate/deploy/access-control/allowlist-general.md [hunk 2]
```diff
@@ -48,6 +48,21 @@ To interact with the ACL precompiles using an external library or client, follow
...
+
+Below is the ABI specification for the ACL:
+
+```shell
+function setAdmin(address)
+function setEnabled(address)
+function setNone(address)
+function readAddressList(address) returns (uint256)
+function setListEnabled(bool)
+function getListEnabled() returns (bool)
```
Comment (Stefan-Ethernal) [inline docs/edge/operate/deploy/access-control/allowlist-general.md:61]: We do not have these two functions, but only these four:
```solidity
function setAdmin(address)
function setEnabled(address)
function setNone(address)
function readAddressList(address) returns (uint256)
```

File: docs/edge/operate/deploy/genesis-validators.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ Each validator needs to register themselves on the `CustomSupernetManager` contr
 
 ```bash
 ./polygon-edge polybft register-validator --data-dir ./test-chain-1 \
---supernet-manager $(cat genesis.json | jq -r '.params.engine.polybft.bridge.stakeManagerAddr')
+  --supernet-manager $(cat genesis.json | jq -r '.params.engine.polybft.bridge.stakeManagerAddr')
```
Comment (Stefan-Ethernal) [inline docs/edge/operate/deploy/genesis-validators.md:71]:
```suggestion
  --supernet-manager $(cat genesis.json | jq -r '.params.engine.polybft.bridge.customSupernetManagerAddr')
```

File: docs/edge/operate/deploy/genesis-validators.md [hunk 4]
```diff
@@ -111,9 +111,9 @@ For example, if you are using the Mumbai test network, you can obtain the addres
 
 ```bash
 curl <mumbai-rpc-endpoint> \
--X POST \
--H "Content-Type: application/json" \
---data '{"jsonrpc":"2.0","method":"eth_contractAddress","params":["MaticToken"],"id":1}'
+  -X POST \
+  -H "Content-Type: application/json" \
+  --data '{"jsonrpc":"2.0","method":"eth_contractAddress","params":["MaticToken"],"id":1}'
```
Comment (Stefan-Ethernal) [inline docs/edge/operate/deploy/genesis-validators.md:116]: AFAIK, there is no "eth_contractAddress" JSON RPC endpoint. Instead given address must be known in advance.

File: docs/edge/operate/deploy/genesis.md [hunk 5]
```diff
@@ -30,142 +30,96 @@ To create the chain configuration, we use the `polygon-edge genesis` command, wh
...
-    --consensus polybft \
-    --burn-contract 100:0x1234567890ABCDEF1234567890ABCDEF12345678
-```
-
-</details>
+| Consideration | Description | Details |
+| --- | --- | --- |
+| **Difference between `chain-id` and `supernet-id`** | - `chain-id`: Unique identifier for a childchain.<br/>- `supernet-id`: Identifier populated during registration with `StakeManager`. | Users can assign a custom `chain-id` via the genesis command. The `supernet-id` value is derived from the `rootchain deploy` command. |
+| **Create a Native Token and Premine** | Configure the native token and premine specific accounts. | - `--premine`: Specify premined accounts and balances.<br/>- `--native-token-config`: Configure the native token's details.<br/>- `--owner` (Note): For mintable native tokens, designates permissions. |
+| **Enable EIP1559** | Enable the London hard fork with specific configurations. | - `--burn-contract`: Specify the burn contract address.<br/>- `--genesis-base-fee`: Set the initial base fee for the genesis block. |
```
Comment (Stefan-Ethernal) [inline docs/edge/operate/deploy/genesis.md:121]: At the moment (as of 1.3.0), we do not have the `--genesis-base-fee` flag exposed. There is a PR that got merged, but only after 1.3.0 (https://github.com/0xPolygon/polygon-edge/pull/1890), so for 1.3.0 it is not possible to parameterize the base fee in the genesis time.

File: docs/edge/operate/deploy/genesis-validators.md (hunk 4 above)
Comment (DannyS03) [reply docs/edge/operate/deploy/genesis-validators.md:116]: updated

File: docs/edge/operate/deploy/genesis.md (hunk 5 above)
Comment (DannyS03) [reply docs/edge/operate/deploy/genesis.md]: Addressed

File: docs/edge/operate/deploy/genesis-validators.md [hunk 6]
```diff
@@ -107,13 +107,13 @@ If you currently hold MATIC tokens, you can convert them to WMATIC through vario
...
 
 ```bash
 curl <mumbai-rpc-endpoint> \
--X POST \
--H "Content-Type: application/json" \
---data '{"jsonrpc":"2.0","method":"eth_contractAddress","params":["MaticToken"],"id":1}'
+  -X POST \
+  -H "Content-Type: application/json" \
+  --data '{"jsonrpc":"2.0","method":"eth_call","params":[{"to":"<token-contract-address>","data":"0x06fdde03"},"latest"],"id":1}'
 ```
```
Comment (Stefan-Ethernal) [inline docs/edge/operate/deploy/genesis-validators.md:117]: I don't get it, what is the purpose of this paragraph? What exactly you are suggesting to be extracted this way?

The address of WMATIC should be known in advance for each network (Mumbai, POS mainnet etc.) and to best of my knowledge it is not possible to discover the address of a contract.

**Review by Stefan-Ethernal — APPROVED** [review]
LGTM in general

File: docs/edge/operate/deploy/genesis-validators.md (hunk 6 above)
Comment (DannyS03) [reply docs/edge/operate/deploy/genesis-validators.md:117]: That's true... trying to remember the context behind this section. It was part of a documentation update of the past, so I won't fold this PR up and do a fast follow to address this section.

**Review by EmpieichO — APPROVED** [review]

**Pull request merged**